cmd/talosctl/pkg/talos/action/tracker.go

```diff
@@ -121,19 +119,18 @@ func WithDebug(debug bool) TrackerOption {
 
 // NewTracker creates a new Tracker.
 func NewTracker(
-	cliContext *global.Args,
+	clientExecutor ClientExecutor,
 	expectedEventFn func(event client.EventResult) bool,
 	actionFn func(ctx context.Context, c *client.Client) (string, error),
 	opts ...TrackerOption,
 ) *Tracker {
 	tracker := Tracker{
 		expectedEventFn:          expectedEventFn,
 		actionFn:                 actionFn,
-		nodeToLatestStatusUpdate: make(map[string]reporter.Update, len(cliContext.Nodes)),
+		nodeToLatestStatusUpdate: make(map[string]reporter.Update, len(clientExecutor.NodeList())),
 		reporter:                 reporter.New(),
 		reportCh:                 make(chan nodeUpdate),
-		isTerminal:               isatty.IsTerminal(os.Stderr.Fd()),
```

Comment:
this shouldn't be dropped? or we should allow to override it via an option?

Reply:
we could, this has no effect on the provider, so i ignored, I could make it an option

Reply:
but it shouldn't be removed either way

Reply:
oh, that was my mistake, forgot to add it back